docs/howto/openstack/nova/new-vgpu-server.md

```diff
@@ -0,0 +1,210 @@
+# Creating new servers with virtual GPU support
+
+In {{brand}}, you can instantiate virtual machines --- or simply
+_servers_ --- with virtual GPU (vGPU) support. For that, you may use the
+{{gui}} or the OpenStack CLI. Following both approaches, let us
+demonstrate the creation of a new server with vGPU support.
+
+## Prerequisites
+
+Although not necessary, it certainly helps a lot to have some
+familiarity with [creating servers in
+{{brand}}](/howto/openstack/nova/new-server). Additionally, if you
```

Comment:
Don't use absolute links, and do make use of Mkdocs' magic linking ability referencing source Markdown files, like so:

```suggestion
{{brand}}](new-server.md). Additionally, if you
```

Reply:
Done (and yes, that's magical).